servers/rendering/renderer_rd/effects_rd.cpp

```diff
@@ -2540,21 +2538,27 @@ EffectsRD::~EffectsRD() {
 		cubemap_downsampler.compute_shader.version_free(cubemap_downsampler.shader_version);
 		filter.compute_shader.version_free(filter.shader_version);
 	}
-	copy.shader.version_free(copy.shader_version);
+	if (!prefer_raster_effects) {
+		copy.shader.version_free(copy.shader_version);
```

Comment:
Why not move this into the else up above?

Reply:
done